openshift/system/Dockerfile

```diff
@@ -97,7 +89,7 @@ ENTRYPOINT ["/usr/libexec/catatonit/catatonit", "--", "/opt/system/entrypoint.sh
 CMD ["unicorn", "-c", "config/unicorn.rb", "-E", "${RAILS_ENV}", "config.ru"]
 
 
-FROM base AS sphinx-config
+FROM builder AS sphinx-config
```

Comment:
if this is moved before `porta-prod`, the config can be copied to it directly, no need for a dediacted `porta-prod-with-sphinx-config` stage